druid-shell/src/window.rs

```diff
@@ -65,10 +65,18 @@ impl IdleHandle {
     where
         F: FnOnce(&dyn Any) + Send + 'static,
     {
-        self.0.add_idle(callback)
+        self.0.add_idle_callback(callback)
+    }
+
+    pub fn schedule_idle(&mut self, token: IdleToken) {
+        self.0.add_idle_token(token)
     }
 }
 
+/// A token that uniquely identifies a idle schedule.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Hash)]
+pub struct IdleToken(usize);
```

Comment:
This needs to have some public constructor, probably just `const fn new(raw: usize)`?